.github/workflows/commitlint.yaml

```diff
@@ -0,0 +1,26 @@
+---
```

Comment:
Is there a reason for having an empty YML header…?

Reply:
No reason, removed.